gluten-ut/spark34/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -689,8 +693,10 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude("vectorized reader: required array with optional elements")
     .exclude("vectorized reader: required array with legacy format")
     .exclude("SPARK-36726: test incorrect Parquet row group file offset")
+    // add support in native reader
     .exclude("SPARK-41096: FIXED_LEN_BYTE_ARRAY support")
     .exclude("SPARK-40128 read DELTA_LENGTH_BYTE_ARRAY encoded strings")
+    // Timestamp is read as INT96
```

Comment:
The spark 34 test is copied from spark 33. Could you please help investigate why the test works in Spark 33 but doesn't work in Spark 34?

Reply:
This test is also a new one in spark 3.4 and there already exists a excluded test in this file for the same reason. Once int64 support gets added these tests can be included.